src/View/ViewableData.php

```diff
@@ -439,10 +441,13 @@ public function castingClass($field)
      *
      * @param string $field
      * @return string 'xml'|'raw'
+     * @deprecated 5.4.0 Will be removed without equivalent functionality to replace it.
      */
     public function escapeTypeForField($field)
```

Comment:
Only one use of this in core, so no need to replace it. If we do need this in the future it can be added to the new `CastingService`.